Vizzuality/heco-invest
# feat: New API endpoint for Project update

New API endpoint for updating existing Project.

## Testing instructions

Open rswag and try to update existing Project of your choice. Make sure that you are ProjectDeveloper who created Project.

## Tracking

https://vizzuality.atlassian.net/browse/LET-390
